src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -430,50 +431,23 @@ string relationshipEntity
         }
 
         /// <summary>
-        /// Test method to check that an exception is thrown when relationship exists between the wrong pair. i.e.,
-        /// LinkingSourceField OR SourceFields are null and relationship between target and linking object exists, no relationship between source and linking object
-        /// LinkingTargetField OR TargetField are null but relationship between source and linking object exists, no relationship between target and linking object
-        /// Also Test when the foreignKeyPair exist for correct pair, no exception is thrown. i.e.,
-        /// LinkingSourceField OR SourceFields are null but relationship between source and linking object exists, no relationship between target and linking object
-        /// LinkingTargetField OR TargetField are null but relationship between target and linking object exists, no relationship between source and linking object
+        /// Test method to check that an exception is thrown when LinkingObject is null
+        /// and SourceFields and TargetFields are null in the config.
+        /// And the foreignKey pair between source and target is not defined in the database as well.
+        /// Also verify that after adding foreignKeyPair in the Database, no exception is thrown.
```

Comment:
adding ForeignKeyPair for which tables? between source and target? between Source/Linking Object and Target/Linking Object?